weed/shell/command_volume_check_disk.go

```diff
@@ -129,6 +130,12 @@ func (c *commandVolumeCheckDisk) doVolumeCheckDisk(subtrahend, minuend *needle_m
 	})
 
 	fmt.Fprintf(writer, "volume %d %s has %d entries, %s missed %d entries\n", source.info.Id, source.location.dataNode.Id, counter, target.location.dataNode.Id, len(missingNeedles))
+	missingNeedlesFraction := float64(len(missingNeedles)) / float64(counter)
+	if missingNeedlesFraction > nonRepairThreshold {
+		return fmt.Errorf(
+			"failed to start repair volume %d, percentage of missing keys is greater than the threshold: %.2f > %.2f",
```

Comment:
@kmlebedev if just skipping why  "failed to start repair volume",  maybe  "skipping repair volume"
or did I misunderstood?

Reply:
> @kmlebedev if just skipping why "failed to start repair volume", maybe "skipping repair volume"
> or did I misunderstood?
Yes, check command exits with this error